gatsby-node.js

```diff
@@ -275,18 +293,12 @@ exports.createSchemaCustomization = ({ actions }) => {
       apiNode: Candidate @link(by: "ID" from: "id")
     }
 
-    type MeasuresJson implements Node {
-      electionDate: String!
-      title: String!
-      description: String!
-      ballotLanguage: String!
-    }
-
```

Comment:
Are we going to delete the JSON files entirely? That's fine if we're going to get all the data we need from the API, but I was under the impression we'd need to have some client-side data for the measure description, etc. If we no longer need that, let's also delete the json files under /src/data

Reply:
I didn't realize that was for the JSON; I'll put it back in.